integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/WildFlyIT.java

```diff
@@ -50,13 +50,10 @@ public static Iterable<Object[]> data() {
         return Arrays.asList(new Object[][]{
             {"8.2.1.Final"},
             {"9.0.0.Final"},
-            {"10.0.0.Final"},
             {"11.0.0.Final"},
-            {"12.0.0.Final"},
             {"13.0.0.Final"},
-            {"14.0.0.Final"},
             {"15.0.0.Final"},
-            {"16.0.0.Final"}
+            {"25.0.0.Final"}
```

Comment:
[For Reviewer]
I removed a few inbetween versions as they are IMO not that useful to get confidence. I decided to remove them to not further increase the testing time.